internal/output/html/script.js

```diff
@@ -252,16 +239,16 @@ function applyLayerFilter(selectedLayerID) {
   tableRows.forEach(row => {
     const rowLayerID = row.getAttribute("data-layer");
     if (selectedLayerID !== "all" && rowLayerID !== selectedLayerID) {
-      const packageDetails = document.getElementById(row.id + "-details");
+      const packageDetails = document.getElementById(`${row.id}-details`);
       const vulnElements = packageDetails.querySelectorAll(".vuln-tr");
       vulnElements.forEach(vuln => {
-        vuln.classList.toggle("hide-block", true);
+        vuln.classList.add("hide-block");
```

Comment:
Oh wow, TIL I've been over complicating this the whole time, I thought just using add will duplicate the class if it already exists, but apparently not: https://stackoverflow.com/questions/51214589/is-there-any-harm-in-adding-a-class-to-an-element-that-already-has-that-class-us

They should have named this classSet haha.